biojava-alignment/src/main/java/org/biojava/nbio/alignment/SimpleGapPenalty.java

```diff
@@ -30,9 +32,13 @@
  *
  * @author Mark Chapman
  */
-public class SimpleGapPenalty implements GapPenalty {
+public class SimpleGapPenalty implements GapPenalty, Serializable {
```

Comment:
Thanks for this. I'm afraid it's necessary in this case. But yes I agree with your caution.